### DISCUSSION TITLE
Unhandled Promise Rejection: invalid dns enhanced-mode: redir-host

### DISCUSSION THREAD

**Discussion by bobbyaaaaaa** [zzzgydi/clash-verge#461] [Q&A]
从路径解析到节点时总报这个错 proxie接口返回的东西都是空
<img width="792" alt="截屏2023-03-20 09 33 02" src="https://user-images.githubusercontent.com/127934935/226226984-8b976c5a-0161-4d95-b1ac-2f10b4c57b07.png">
<img width="778" alt="截屏2023-03-20 09 35 08" src="https://user-images.githubusercontent.com/127934935/226227049-b12dc4f0-1112-4678-9cb2-3838447931e3.png">

**Comment by zzzgydi** [comment]
现在clash premium核心的dns已经不支持redir-host模式了，改成fakeip吧
